### PR TITLE
Top positive and negative words from Twitter

### PR DESCRIPTION
**Pull request by mjvallone**
This [PR](https://github.com/mjvallone/companies_review/pull/7) should be approved first since the current one is based on it.

### CODE REVIEW

File: data_transformation.py [hunk 1]
```diff
@@ -1,21 +1,134 @@
...
+    data['emojis'] = data['processed_text'].apply(lambda x: count_emoji(x))
+
+
+def get_sentiments(data):
+    sentiment_pipeline = pipeline(model=MODEL_PATH)
 
+    data['output_clean'] = data['processed_text'].apply(lambda tweet: sentiment_pipeline(tweet))
+    data['sentiment_label'] = data['output_clean'].apply(lambda dic: dic[0]['label'])
+    data['sentiment_score'] = data['output_clean'].apply(lambda dic: dic[0]['score'])
+    #TODO drop output_clean?
```
Comment (mjvallone) [inline data_transformation.py]: do we need it later? any thoughts @RomiLanda ?

File: data_transformation.py [hunk 2]
```diff
@@ -1,21 +1,134 @@
...
+def get_ranked_tweets(data, stop_words, filter_condition):
+    tweets = data['processed_text'][data['sentiment_label']==filter_condition]
+    df_tw = sort_df_freq_words(tweets, stop_words)
+    df_tw = df_tw[0:10] #FIXME how many should we show?
+    df_tw['positive'] = True if filter_condition=='positive' else False
+    return df_tw
+
+
+def get_top_pos_neg_tokens(company_name, data):
+    # FIXME from where "https", "co", "RT" came? should we remove them when cleaning data?
```
Comment (mjvallone) [inline data_transformation.py:120]: this is for you :)  @RomiLanda

File: data_transformation.py (hunk 1 above)
Comment (RomiLanda) [reply data_transformation.py]: I think we could eliminate it, it's redundant

File: data_transformation.py (hunk 2 above)
Comment (RomiLanda) [reply data_transformation.py:120]: I am going to check out "co" and "RT", but in the **process _data**  we remove HTTPS with this code: `text = re.sub('((www\.[^\s]+)|(https?://[^\s]+))', ' ', text)`.

**Review by RomiLanda — APPROVED** [review]

File: data_transformation.py (hunk 2 above)
Comment (mjvallone) [reply data_transformation.py:120]: k, I'll leave that comment so you can work on it when ypu find time

**Pull request merged**